pycm/pycm_obj.py

```diff
@@ -828,3 +828,32 @@ def position(self):
                             positions[label]['TN'].append(index)
             self.positions = positions
         return self.positions
+
+    def to_array(self, normalized=False, one_vs_all=False, class_name=None):
+        """
+        Return the confusion matrix in form of  a numpy array.
+
+        :param normalized: A flag for getting normalized confusion matrix
+        :type normalized: bool
+        :param one_vs_all : One-Vs-All mode flag
+        :type one_vs_all : bool
+        :param class_name : target class name for One-Vs-All mode
+        :type class_name : any valid type
+        :return: confusion matrix as a numpy.ndarray
+        """
+        classes = self.classes
+        classes.sort()
+        if normalized:
+            table = self.normalized_table
+        else:
+            table = self.table
```

Comment:
I think it would be better to move this line to outside of the else (before if).